src/NuGet.Core/NuGet.Commands/RestoreCommand/Diagnostics/UnexpectedDependencyMessages.cs

```diff
@@ -210,11 +230,6 @@ public static bool DependencyRangeHasMissingExactMatch(ResolvedDependencyKey dep
         /// </summary>
         public static bool HasMissingLowerBound(VersionRange range)
         {
-            if (range == null)
```

Comment:
Is this needed with the change at 198?

Reply:
I think this null check is not needed, or you are referring to the function?

Reply:
I was referring to the null  check.

Reply:
Yes, I deleted this so we don't display two NU1604 error messages